packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -257,14 +266,55 @@ const TransactionHeader = memo(
           width={20}
           onSelect={e => dispatchSelected({ type: 'select-all', event: e })}
         />
-        <Cell value="Date" width={110} />
-        {showAccount && <Cell value="Account" width="flex" />}
-        <Cell value="Payee" width="flex" />
+        <Cell
+          value="Date"
+          id="date"
+          width={110}
+          icon={field === 'date' && ascDesc}
+          onClick={e => onSortTable(e)}
+        />
+        {showAccount && (
+          <Cell
+            value="Account"
+            id="account"
+            width="flex"
+            icon={field === 'account' && ascDesc}
+            onClick={e => onSortTable(e)}
+          />
+        )}
+        <Cell
+          value="Payee"
+          id="payee"
+          width="flex"
+          icon={field === 'payee' && ascDesc}
+          onClick={e => onSortTable(e)}
+        />
         <Cell value="Notes" width="flex" />
-        {showCategory && <Cell value="Category" width="flex" />}
-        <Cell value="Payment" width={80} textAlign="right" />
-        <Cell value="Deposit" width={80} textAlign="right" />
-        {showBalance && <Cell value="Balance" width={88} textAlign="right" />}
+        {showCategory && (
+          <Cell
+            value="Category"
+            id="category"
+            width="flex"
+            icon={field === 'category' && ascDesc}
+            onClick={e => onSortTable(e)}
+          />
+        )}
+        <Cell
+          value="Payment"
+          id="payment"
+          width={90}
+          textAlign="flex"
+          icon={field === 'amount' && ascDesc}
+          onClick={e => onSortTable(e)}
+        />
+        <Cell
+          value="Deposit"
+          id="deposit"
+          width={85}
+          textAlign="flex"
+          onClick={e => onSortTable(e)}
+        />
```

Comment:
It felt a bit weird that I could sort by payment but not deposit — maybe it would make sense to move the indicator to the Deposit icon so the two columns feel more joined than separated by the icon?